src/utils/index.ts

```diff
@@ -11,7 +11,11 @@ export const hexToRGBA = (hex: string, alpha: string | number) => {
 };
 
 export const isEmpty = (value: any) =>
-    value === undefined || value === null || value === '' || value === {} || value === [];
+    value === undefined ||
+    value === null ||
+    (typeof value === 'string' && value.trim() === '') ||
+    (Array.isArray(value) && value.length === 0) ||
+    (typeof value === 'object' && Object.keys(value).length === 0);
```

Comment:
Can you handle these edge cases as well ?

<img width="464" alt="Screenshot 2022-07-08 at 8 48 57 PM" src="https://user-images.githubusercontent.com/17466642/178021657-27dcf275-81d4-4fe3-8c8b-924a97abf6f5.png">

Reply:
Added those cases. Just curious: Why aren't we using `lodash.isEmpty()`?

Reply:
> Why aren't we using lodash.isEmpty()

![image](https://user-images.githubusercontent.com/17466642/178132673-f2581e72-b446-4c7a-b991-eb4bf8156bc8.png)